kubectl-fdb/cmd/root.go

```diff
@@ -23,12 +23,13 @@ package cmd
 import (
 	"bufio"
 	"fmt"
-	"github.com/fatih/color"
 	"log"
 	"math/rand"
 	"os"
 	"time"
 
+	"github.com/fatih/color"
+
```

Comment:
Any reason to change the order of the imports?

Reply:
nope. my VSCode seems changing this automatically.